PIL/Image.py

```diff
@@ -538,12 +573,15 @@ def _new(self, im):
 
     # Context Manager Support
     def __enter__(self):
+        # type: () -> Image
         return self
 
     def __exit__(self, *args):
+        # type: (*Any) -> None
```

Comment:
There's a specific set of what can be called on a context manager. We can change the signature to that, as it's the only way that this should ever get called.